packages/mpv/mpv.conf

```diff
@@ -19,4 +19,4 @@ ao=opensles
 # audio-format=s16
 
 # Disable Video Decode and Output. Termux doesn't support video output (with the exception of "tct").
-vid=no
+vo=tct
```

Comment:
As mentioned in the manpage[^1], `vo=tct` does not perform particularly well so I would be hesitant to make it a default for our headless MPV package.
[^1]: https://man.archlinux.org/man/mpv.1#tct